CorsixTH/Lua/world.lua

```diff
@@ -1636,7 +1636,13 @@ function World:getPath(x, y, dest_x, dest_y)
   return self.pathfinder:findPath(x, y, dest_x, dest_y)
 end
 
-function World:getIdleTile(x, y, idx)
+-- Find an tile for idling.
+-- !param x X coordinate of the queue position.
+-- !param y Y coordinate of the queue position.
+-- !param idx Randomization factor for the idle tile, returns Nth candidate tile.
+-- !param parcel_id Optional parcel of the indoor returned idling tile.
+-- !return Position of an idle tile if it exists.
+function World:getIdleTile(x, y, idx, parcel_id)
```

Comment:
It `might` even be better to do `parcel_id = parcel_id or 0` here to make it clear nil parcel_ids are not a real thing?